src/components/ui/calendar.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import { ChevronLeft, ChevronRight } from "lucide-react";
+import { DayPicker } from "react-day-picker";
+
+import { cn } from "@/lib/utils";
+import { buttonVariants } from "@/components/ui/button";
+
+export type CalendarProps = React.ComponentProps<typeof DayPicker>;
+
+function Calendar({ className, classNames, showOutsideDays = true, ...props }: CalendarProps) {
```

Comment:
Jestli máš s těma props problémy tady, jak jsi psala, tak to se občas stává, protože Shadcn nedodržuje úplně striktní eslist. Když ti to nepůjde vyřešit, tak si tohle klidně hoď do ChatGPT, ať ti to ten typ upravý, nebo tam dej `// @eslint-ignore-next-line` a ten error ti zmizí.